dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/event/RumEventSerializer.kt

```diff
@@ -72,6 +75,28 @@ internal class RumEventSerializer : Serializer<RumEvent> {
         }
     }
 
+    private fun addNetworkInfo(model: RumEvent, root: JsonObject) {
+        val info = model.networkInfo
+        if (info != null) {
+            root.addProperty(RumAttributes.NETWORK_CONNECTIVITY, info.connectivity.serialized)
+            if (!info.carrierName.isNullOrBlank()) {
+                root.addProperty(RumAttributes.NETWORK_CARRIER_NAME, info.carrierName)
+            }
+            if (info.carrierId >= 0) {
+                root.addProperty(RumAttributes.NETWORK_CARRIER_ID, info.carrierId)
+            }
+            if (info.upKbps >= 0) {
+                root.addProperty(RumAttributes.NETWORK_UP_KBPS, info.upKbps)
+            }
+            if (info.downKbps >= 0) {
+                root.addProperty(RumAttributes.NETWORK_DOWN_KBPS, info.downKbps)
+            }
+            if (info.strength > Int.MIN_VALUE) {
```

Comment:
shouldn't be `>0` here ?

Reply:
Not necessarily, the info strength is dependant on the kind of network and some can use a negative value.